sphere-models/src/main/java/io/sphere/sdk/carts/Cart.java

```diff
@@ -79,4 +93,8 @@ public String toString() {
 
     @Override
     List<DiscountCodeInfo> getDiscountCodes();
+
+    @Nullable
+    @Override
+    CustomFields getCustom();
```

Comment:
I know it is kinda weird in the code, but the goal is to have a good json and predicate support:
- json:
```
{
  [...]
  "custom": {
    "fieldA": "valueA"
  }
}
```

and the predicate `myobject.custom.fieldA = "valueA" `